lib/db.ts

```diff
@@ -127,64 +153,76 @@ const data = {
-        EXTENDED_DESCRIPTION: "```Use this module to find the lyrics of a song by using```  *.lyrics*  ```command.```",
+        EXTENDED_DESCRIPTION:
+            "```Use this module to find the lyrics of a song by using```  *.lyrics*  ```command.```",
         NO_ARG: "```Please enter the song name.```",
         NOT_FOUND: "```Song not found !```",
-        PROCESSING: "```Searching. Please wait....```"
+        PROCESSING: "```Searching. Please wait....```",
     },
     meaning: {
         DESCRIPTION: "Find meaning of a word in dictionary.",
-        EXTENDED_DESCRIPTION: "Find meaning of a word in dictionary by using .meaning <word>.",
+        EXTENDED_DESCRIPTION:
+            "Find meaning of a word in dictionary by using .meaning <word>.",
         NO_ARG: "```Please enter a word.```",
         NOT_FOUND: "```Word not found in dictionary!```",
     },
     mute: {
         DESCRIPTION: "Mute group chat for a specified time.",
-        EXTENDED_DESCRIPTION: "Mute non-admin members of a group. You can even specify the duration using s, m or h.\n\nFor example:\n .mute 15 m   will change chat permissions to admin-only for 15 minutes.",
+        EXTENDED_DESCRIPTION:
+            "Mute non-admin members of a group. You can even specify the duration using s, m or h.\n\nFor example:\n .mute 15 m   will change chat permissions to admin-only for 15 minutes.",
         NOT_GROUP_CHAT:
             "*.mute*  ```command is only applicable in a group chat.```",
         NOT_ADMIN:
             "```Sorry, dont have the permission to do so since I am not an admin.```",
         CHAT_ADMIN_ONLY: "```Chat permissions changed to```  *admin only*.",
         MENTION_DURATION:
             "```Please mention how long you want to mute the chat. For example,```\n*.mute 10 s*  ```to mute for 10 seconds.```",
-        CHAT_ALL_MEMBERS:
-            "```Chat permissions changed to```  *all group members*."
+        CHAT_ALL_MEMBERS: "```Chat permissions changed to```  *all group members*.",
     },
     neko: {
         DESCRIPTION: "Copy your text to nekobin",
         EXTENDED_DESCRIPTION:
             "```Use this module to paste your text to a pastebin (NEKOBIN). Enter text with the command```  *.neko* .",
         ENTER_TEXT: "```Please enter a text message along with the command.```",
         TRY_LATER: "```Too many tries. Please try again later.```",
-        PROCESSING: "```Pasting text to nekobin. Please wait...```"
+        PROCESSING: "```Pasting text to nekobin. Please wait...```",
+    },
+    news: {
+        DESCRIPTION: "Fetch news in a mobile readable format",
+        EXTENDED_DESCRIPTION:
+            "```Use this module to fetch news in mobile readable format from around 1500 publications. Enter text with the command```  *.news* .\n\nTo search for publications, use:-\n```.news search``` _search term_\n\nTo fetch news, use:-\n```.news fetch``` _Publication Name_",
+        NO_COMMMAND: "```Please provide an action.```",
```

Comment:
```suggestion
        NO_COMMMAND: "```No arguments supplied. Use``` *.help news* ```for info on how to use the command.```",
```